api/getBlogs/data/Touroperators.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "Your Guide to Finding the Best Tour Operators in Varanasi",
+    "metaDescription": "Looking for the perfect tour guide in Varanasi? Discover how to choose the best tour operators for an unforgettable experience in this holy city.",
+    "pageName": "best-tour-operators-in-varanasi",
+    "description": "Essential tips and insights on selecting the ideal tour operator in Varanasi to ensure a memorable journey through this sacred city.",
+    "keywords": ["Varanasi tour operators", "tour guide Varanasi", "Ganga Aarti", "boat rides Ganges", "Varanasi tours"],
+    "images": {
+      "id": "sHWTwOyZ5mflEoXP",
+      "description": "",
+      "file": ["../images/touroperator.png"]
```

Comment:
What is size of image? Is it under 70 KB?

Reply:
149 KB